modules/st2-auto-form/tests/test-array.js

```diff
@@ -30,11 +30,11 @@ describe('AutoForm ArrayField', () => {
 
     const c = new TestComponent(<ArrayField {...props} />);
 
-    c.makeChange('t,e,s,t');
+    c.makeChange('["t","e","s","t"]');
```

Comment:
Do we still handle case without brackes in the same manner? Is this even desired now?